kalite/i18n/__init__.py

```diff
@@ -442,3 +444,10 @@ def set_request_language(request, lang_code):
 
     request.language = lcode_to_ietf(lang_code)
     translation.activate(request.language)
+
+@contextmanager
```

Comment:
@aronasorman Here is the context block for translation. Have used it as a with statement in all relevant places in code.